test/tables.jl

```diff
@@ -70,7 +70,7 @@ Base.propertynames(d::DuplicateNamesColumnTable) = (:a, :a, :b)
         @test @inferred(Tables.materializer(table)(Tables.columns(table))) isa typeof(table)
 
         row = first(Tables.rows(table))
-        @test propertynames(row) == (:a, :b)
+        @test collect(propertynames(row)) == [:a, :b]
```

Comment:
E.g. this is the point to discuss as I have raised. Now depending on if you pass e.g. `df` or `eachrow(df)` to `Tables.rows` you get a different type of `row` and in consequence a different type of `propertynames` return value. Is this OK to have.?

Reply:
Actually I have made https://github.com/JuliaData/DataFrames.jl/pull/2056 to make sure we do not have to change this line as `propertynames` now always returns a `Tuple`.